tgcf
Running latest tgcf version 1.1.8


[08/06/23 04:02:52] INFO     Verbosity turned on! This is suitable for debugging    cli.py:57
                    INFO     cryptg detected, it will be used for encryption        aes.py:19
                    INFO     tgcf.config.json detected!                         config.py:113
                    WARNING  You have not set a password to protect the web     config.py:278
                             access to tgcf.                                                 
                             The default password `tgcf` is used.                            
                    INFO     config.py got executed                             config.py:283
                    INFO     using session string                               config.py:236
                    INFO     Connecting to 149.154.167.91:443/TcpFull... mtprotosender.py:226
                    INFO     Connection to 149.154.167.91:443/TcpFull    mtprotosender.py:277
                             complete!                                                       
[08/06/23 04:02:53] INFO     config.is_bot=False                                  live.py:149
                    INFO     Command prefix is . for userbot and / for bot    live_bot.py:136
                    INFO     Loaded admins are []                               config.py:230
                    INFO     Added event handler for new                          live.py:160
                    INFO     Added event handler for edited                       live.py:160
                    INFO     Added event handler for start                        live.py:160
                    INFO     Added event handler for forward                      live.py:160
                    INFO     Added event handler for remove                       live.py:160
                    INFO     Added event handler for style                        live.py:160
                    INFO     Added event handler for help                         live.py:160
                    INFO     Forward                                            config.py:196
                    INFO     Dest: -1001932245430/2                             config.py:201
                    INFO     Reply_to: 2                                        config.py:207
                    INFO     Dest: -1001760549411                               config.py:201
                    INFO     Forward.dest: [-1001932245430, -1001760549411]     config.py:212
                    INFO     Forward.reply_to: 0                                config.py:213
                    INFO     Destination object: dest=-1001760549411 reply_to=0 config.py:221
                    INFO     Forward                                            config.py:196
                    INFO     Dest: -1001932245430/10                            config.py:201
                    INFO     Reply_to: 10                                       config.py:207
                    INFO     Dest: -1001760549411                               config.py:201
                    INFO     Forward.dest: [-1001932245430, -1001760549411]     config.py:212
                    INFO     Forward.reply_to: 0                                config.py:213
                    INFO     Destination object: dest=-1001760549411 reply_to=0 config.py:221
                    INFO     Forward                                            config.py:196
                    INFO     Dest: -1001932245430/9                             config.py:201
                    INFO     Reply_to: 9                                        config.py:207
                    INFO     Dest: -1001760549411                               config.py:201
                    INFO     Forward.dest: [-1001932245430, -1001760549411]     config.py:212
                    INFO     Forward.reply_to: 0                                config.py:213
                    INFO     Destination object: dest=-1001760549411 reply_to=0 config.py:221
                    INFO     Forward                                            config.py:196
                    INFO     Dest: -1001932245430/8                             config.py:201
                    INFO     Reply_to: 8                                        config.py:207
                    INFO     Dest: -1001760549411                               config.py:201
                    INFO     Forward.dest: [-1001932245430, -1001760549411]     config.py:212
                    INFO     Forward.reply_to: 0                                config.py:213
                    INFO     Destination object: dest=-1001760549411 reply_to=0 config.py:221
                    INFO     Forward                                            config.py:196
                    INFO     Dest: -1001932245430/4                             config.py:201
                    INFO     Reply_to: 4                                        config.py:207
                    INFO     Dest: -1001760549411                               config.py:201
                    INFO     Forward.dest: [-1001932245430, -1001760549411]     config.py:212
                    INFO     Forward.reply_to: 0                                config.py:213
                    INFO     Destination object: dest=-1001760549411 reply_to=0 config.py:221
                    INFO     Forward                                            config.py:196
                    INFO     Dest: -1001932245430/7                             config.py:201
                    INFO     Reply_to: 7                                        config.py:207
                    INFO     Dest: -1001760549411                               config.py:201
                    INFO     Forward.dest: [-1001932245430, -1001760549411]     config.py:212
                    INFO     Forward.reply_to: 0                                config.py:213
                    INFO     Destination object: dest=-1001760549411 reply_to=0 config.py:22[08/06/23 09:24:07] INFO     The server closed the connection                 connection.py:335
                    INFO     <class 'ConnectionResetError'> during            connection.py:279
                             disconnect: [Errno 54] Connection reset by peer                   
                    INFO     Connection closed while receiving data        mtprotosender.py:507
                    INFO     Closing current connection to begin           mtprotosender.py:350
                             reconnect...                                                      
                    INFO     <class 'ConnectionResetError'> during            connection.py:279
                             disconnect: [Errno 54] Connection reset by peer                   
                    INFO     Connecting to 149.154.167.91:443/TcpFull...   mtprotosender.py:226
                    INFO     Connection to 149.154.167.91:443/TcpFull      mtprotosender.py:277
                             complete!                                                         
[08/06/23 11:16:35] INFO     The server closed the connection                 connection.py:335
                    INFO     <class 'ConnectionResetError'> during            connection.py:279
                             disconnect: [Errno 54] Connection reset by peer                   
                    INFO     Connection closed while receiving data        mtprotosender.py:507
                    INFO     Closing current connection to begin           mtprotosender.py:350
                             reconnect...                                                      
                    INFO     <class 'ConnectionResetError'> during            connection.py:279
                             disconnect: [Errno 54] Connection reset by peer                   
                    INFO     Connecting to 149.154.167.91:443/TcpFull...   mtprotosender.py:226
                    INFO     Connection to 149.154.167.91:443/TcpFull      mtprotosender.py:277
                             complete!                                                         
[08/06/23 15:02:00] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connecting to 149.154.167.91:443/TcpFull... mtprotosender.py:226
[08/06/23 15:02:01] INFO     Connection to 149.154.167.91:443/TcpFull    mtprotosender.py:277
                             complete!                                                       
[08/06/23 15:17:29] WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264178655781656577, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264178680039069697, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264178684059367425, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264179660505216001, ignoring                [08/06/23 15:17:39] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    [08/06/23 16:38:45] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by   [08/06/23 16:39:00] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    [08/06/23 16:49:46] INFO     Got difference for account updates                updates.py:302
[08/06/23 16:49:47] INFO     Got difference for channel 1932245430 updates     updates.py:371
[08/06/23 17:21:30] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by   [08/06/23 17:25:38] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    [08/06/23 17:35:25] INFO     Got difference for account updates                updates.py:302
[08/06/23 17:39:39] INFO     Account is now banned in 1628126888 so we can no  updates.py:351
                             longer fetch updates from it                                    
[08/06/23 17:50:26] INFO     Account is now banned in 1628257738 so we can no  updates.py:351
                             longer fetch updates from it                                    
[08/06/23 20:17:06] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connecting to 149.154.167.91:443/TcpFull... mtprotosender.py:226
                    INFO     Connection to 149.154.167.91:443/TcpFull    mtprotosender.py:277
                             complete!                                                       
[08/06/23 22:47:21] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connecting to 149.154.167.91:443/TcpFull... mtprotosender.py:226
                    INFO     Connection to 149.154.167.91:443/TcpFull    mtprotosender.py:277
                             complete!                                                       
[08/07/23 00:55:57] INFO     The server closed the connection               connection.py:335
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connection closed while receiving data      mtprotosender.py:507
                    INFO     Closing current connection to begin         mtprotosender.py:350
                             reconnect...                                                    
                    INFO     <class 'ConnectionResetError'> during          connection.py:279
                             disconnect: [Errno 54] Connection reset by                      
                             peer                                                            
                    INFO     Connecting to 149.154.167.91:443/TcpFull... mtprotosender.py:226
                    INFO     Connection to 149.154.167.91:443/TcpFull    mtprotosender.py:277
                             complete!                                                       
[08/07/23 01:07:58] INFO     Got difference for account updates                updates.py:302
                    INFO     Got difference for channel 1710369801 updates     updates.py:371
                    INFO     Got difference for channel 1932245430 updates     updates.py:371
[08/07/23 01:20:45] WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264335164201601025, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264335212123545601, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264335222931344385, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264335224380669953, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264335261769202689, ignoring                                   
                    WARNING  Server sent a very old message with ID       mtprotostate.py:211
                             7264336071884417025, ignoring                                   
INFO     Connection to 149.154.167.91:443/TcpFull    mtprotosender.py:277
                             complete!                                                       
[08/07/23 00:56:42] INFO     Got difference for account updates                updates.py:302
                    INFO     New message received in -1001534771874                live.py:27
                    INFO     Config.from_to dest:                                  live.py:43
                             [Destination(dest=-1001932245430, reply_to=4),                  
                             Destination(dest=-1001760549411, reply_to=0)]                   
                    INFO     TM.reply_to in live -> None                           live.py:48
                    INFO     Config.from_to inside dest loop: dest=-1001932245430  live.py:59
                             reply_to=4                                                      
                    INFO     Config.from_to.reply_to inside dest loop: 4           live.py:60
                    INFO     TM.reply_to in utils: 4                              utils.py:45
                    INFO     Recipient: dest=-1001932245430 reply_to=4            utils.py:46
                    INFO     New message received in -1001534771874                live.py:27
                    INFO     Config.from_to dest:                                  live.py:43
                             [Destination(dest=-1001932245430, reply_to=4),                  
                             Destination(dest=-1001760549411, reply_to=0)]                   
                    INFO     TM.reply_to in live -> None                           live.py:48
                    INFO     Config.from_to inside dest loop: dest=-1001932245430  live.py:59
                             reply_to=4                                                      
                    INFO     Config.from_to.reply_to inside dest loop: 4           live.py:60
                    INFO     TM.reply_to in utils: 4                              utils.py:45
                    INFO     Recipient: dest=-1001932245430 reply_to=4            utils.py:46
                    INFO     New message received in -1001534771874                live.py:27
                    INFO     Config.from_to dest:                                  live.py:43
                             [Destination(dest=-1001932245430, reply_to=4),                  
                             Destination(dest=-1001760549411, reply_to=0)]                   
                    INFO     TM.reply_to in live -> None                           live.py:48
                    INFO     Config.from_to inside dest loop: dest=-1001932245430  live.py:59
                             reply_to=4                                                      
                    INFO     Config.from_to.reply_to inside dest loop: 4           live.py:60
                    INFO     TM.reply_to in utils: 4                              utils.py:45
                    INFO     Recipient: dest=-1001932245430 reply_to=4            utils.py:46
                    INFO     Got difference for channel 1932245430 updates     updates.py:371
                    INFO     Config.from_to inside dest loop: dest=-1001760549411  live.py:59
                             reply_to=0                                                      
                    INFO     Config.from_to.reply_to inside dest loop: 0           live.py:60
                    INFO     TM.reply_to in utils: 0                              utils.py:45
                    INFO     Recipient: dest=-1001760549411 reply_to=0            utils.py:46
                    INFO     Config.from_to inside dest loop: dest=-1001760549411  live.py:59
                             reply_to=0                                                      
                    INFO     Config.from_to.reply_to inside dest loop: 0           live.py:60
                    INFO     TM.reply_to in utils: 0                              utils.py:45
                    INFO     Recipient: dest=-1001760549411 reply_to=0            utils.py:46
                    INFO     Config.from_to inside dest loop: dest=-1001760549411  live.py:59
                             reply_to=0                                                      
                    INFO     Config.from_to.reply_to inside dest loop: 0           live.py:60
                    INFO     TM.reply_to in utils: 0                              utils.py:45
                    INFO     Recipient: dest=-1001760549411 reply_to=0            utils.py:46
                    INFO     Got difference for channel 1710369801 updates     updates.py:371
[08/07/23 01:22:32] INFO     Got difference for account updates                updates.py:302
                    INFO     Got difference for channel 1781702697 updates     updates.py:371
                    INFO     Got difference for channel 1684058538 updates     updates.py:371
                    INFO     Message edited in -1001684058538                      live.py:79
                    INFO     TM.reply_to in utils: 9                              utils.py:45
                    INFO     Recipient: dest=-1001932245430 reply_to=9            utils.py:46
                    INFO     TM.reply_to in utils: 0                              utils.py:45
                    INFO     Recipient: dest=-1001760549411 reply_to=0            utils.py:46
                    INFO     Got difference for channel 1434142606 updates     updates.py:371
                    INFO     Got difference for channel 1452351575 updates     updates.py:371
